api/src/main/java/org/openmrs/api/impl/OrderServiceImpl.java

```diff
@@ -985,16 +980,16 @@ public OrderType getOrderTypeByConcept(Concept concept) {
 	public List<Concept> getTestSpecimenSources() {
 		return getSetMembersOfConceptSetFromGP(OpenmrsConstants.GP_TEST_SPECIMEN_SOURCES_CONCEPT_UUID);
 	}
-
+	
 	@Override
 	public Concept getNonCodedDrugConcept() {
 		String conceptUuid = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GP_DRUG_ORDER_DRUG_OTHER);
-		if(org.apache.commons.lang.StringUtils.isNotBlank(conceptUuid)){
+		if (org.apache.commons.lang.StringUtils.isNotBlank(conceptUuid)) {
```

Comment:
You don't have to but it would be nice to clean it up, the two method calls are equivalent but the with the second you won't have that 'ugly' fully qualified java class name